posthog-react-native/src/types.ts

```diff
@@ -17,9 +18,43 @@ export type PostHogAutocaptureOptions = {
   ignoreLabels?: string[]
   propsToCapture?: string[]
 
-  // Navigation
+  /**
+   * Capture the screen name (or route name) and properties
+   *
+   * Only used for expo-router, @react-navigation/native and react-native-navigation
+   *
+   * For react-native-navigation, you need to call initReactNativeNavigation before using this option
+   *  See example: https://posthog.com/docs/libraries/react-native#with-react-native-navigation-and-autocapture
+   *
+   * For @react-navigation/native v6 and below, you need to wrap the PostHogProvider within the NavigationContainer
+   *  See example: https://posthog.com/docs/libraries/react-native#with-react-navigationnative-and-autocapture
+   *
+   * For @react-navigation/native v7 and above, because of a library change, you'll need to capture the screens manually and disable this option
```

Comment:
See https://github.com/PostHog/posthog-js-lite/issues/547 and https://github.com/PostHog/posthog-js-lite/issues/560